terraform/errors.go

```diff
@@ -1,16 +1,29 @@
 package terraform
 
+import "fmt"
+
 type CustomError struct {
 	message   string
 	errors    []error
 	userError bool
 }
 
 func (err *CustomError) Error() string {
-	// TODO: include more details here with user information
 	return err.message
 }
 
+func GenerateDebugLogs(err error) string {
+	customError, ok := err.(*CustomError)
+
+	debugging := ""
+	if ok {
+		for _, e := range customError.errors {
+			debugging = fmt.Sprintf("%s\n%s", debugging, e.Error())
```

Comment:
nit: It looks like in the current implementation, we always add an initial newline character.
Could I suggest we instead first aggregate the error messages into a slice with the `for` loop, then we use the `strings.Join` function with the newline character?

Reply:
The newline is purposeful - have a look at the screenshots in https://github.com/snyk/snyk/pull/2769. I want every error to be one line